.gitlab-ci.yml

```diff
@@ -57,6 +57,8 @@ workflow:
       when: never
     - if: '$CI_PIPELINE_SOURCE == "merge_request_event" || $CI_PIPELINE_SOURCE == "external_pull_request_event"'
     - if: '$CI_COMMIT_BRANCH == "main" || $CI_COMMIT_BRANCH == "stages/prod"'
+    - if: '$CI_MERGE_REQUEST_IID || $CI_EXTERNAL_PULL_REQUEST_IID'
```

Comment:
I was surprised to see `IID` instead of `ID` ... but verified it's the real deal! https://docs.gitlab.com/ee/ci/variables/predefined_variables.html